### ISSUE TITLE
Data does not update as expected

### ISSUE THREAD

**Issue by simonjowett** [troykelly/homeassistant-au-nsw-covid#52]
Hi Troy - Whilst the set-up has the update switch in the  "On" position the data never updates.  Below is a paste of an error in the HA logs file.  Any ideas what I'm doing wrong?  I tried deleting and reinstalling.  

Platform RPi - latest versions of H.A

Logger: nswcovid.protocol.statistics
Source: /usr/local/lib/python3.9/site-packages/nswcovid/protocol/statistics.py:145
First occurred: 9:21:39 AM (6 occurrences)
Last logged: 9:21:39 AM

interstate_active could not find selector #ContentHtml1Zone2 > div:nth-child(1) > div > div:nth-child(4) > ul > li:nth-child(3) > span
overseas_active could not find selector #ContentHtml1Zone2 > div:nth-child(1) > div > div:nth-child(4) > ul > li:nth-child(3) > span
hospitalisations_admitted could not find selector #ContentHtml1Zone2 > div:nth-child(1) > div > div:nth-child(7) > ul > li:nth-child(1) > span
hospitalisations_icu could not find selector #ContentHtml1Zone2 > div:nth-child(1) > div > div:nth-child(7) > ul > li:nth-child(2) > span
hospitalisations_ventilation could not find selector #ContentHtml1Zone2 > div:nth-child(1) > div > div:nth-child(7) > ul > li:nth-child(3) > span

**Comment by troykelly** [comment]
Thank's @simonjowett 
They are two (unrelated, but equally annoying) issues
The layout of the page we are scraping data from keeps changing, depending on the day of the week. I'm going to have to work out some more smarts to look for content.

The second issue (the more important one) is that the update event isn't firing for some reason - I've started looking at it - hopefully can fix on the weekend.

**Comment by simonjowett** [comment]
Thanks for responding - don't spoil your weekend trying to resolve it though.  Go to the park and have a picnic with 5 friends instead!    :)

**Comment by DavidFW1960** [comment]
Hey Troy.. any progress on this? There seem to be more and more sensors unknown/unavailable now..
![image](https://user-images.githubusercontent.com/36390762/139352154-ca4bb703-a1ee-4ef3-b877-484352a45698.png)

**Comment by troykelly** [comment]
Sorry @DavidFW1960 - the page has completely changed.
I just started a new role this week and haven't had a second to come up for air.
Hopefully, some relaxing debugging will get this fixed on the weekend.

**Comment by DavidFW1960** [comment]
Yeah no worries mate. Just wanted to touch base as this is getting worse.. Take it easy..
